components/select_team/select_team.jsx

```diff
@@ -273,6 +276,7 @@ export default class SelectTeam extends React.Component {
                     <Link
                         id='createNewTeamLink'
                         to='/create_team'
+                        onClick={() => trackEvent('select_team', 'click_create_team')}
```

Comment:
Nit on introducing anonymous functions here and in `signup_email`, versus putting it in an instance method. I don't think re-renders make a big deal on these pages, so it's non-blocking.